drivers/nrf_wifi/bus_if/bus/spi/inc/spi.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+/**
+ * @brief Header for the Linux SPI bus layer specific structure declarations of the
+ * Wi-Fi driver.
+ */
+
+#ifndef __LINUX_SPI_H__
+#define __LINUX_SPI_H__
```

Comment:
is this Linux stuff?

Reply:
nope, it was ported from Linux and this was a leftover, will fix.